src/background/services/network/models.ts

```diff
@@ -65,4 +67,17 @@ export type AddEthereumChainDisplayData = {
   };
 };
 
+export interface AddNetworkPayload {
```

Comment:
Let's move this definition into the handler's file to ensure we are not reusing it elsewhere in our codebase.